Code/skyrim_ui/src/assets/i18n/fr.json

```diff
@@ -0,0 +1,155 @@
+{
+  "INFO": "DO NOT MODIFY THIS FILE! USE THE overwrite.json FILE TO TRANSLATE THE UI!",
+  "COMPONENT": {
+    "CHAT": {
+      "SEND": "Envoyer",
+      "MESSAGE": "Message",
+      "MESSAGE_TOO_LONG": "Vous ne pouvez pas envoyer un message d'une longueur supérieure à  {{chatMessageLengthLimit}} caractères."
+    },
+    "CONNECT": {
+      "INFO": {
+        "SERVER_ADDRESS": "Connectez-vous à un serveur en utilisant son adresse complète. Utilisez le format  <strong>addresse:port</strong>. Le port par défaut est : <strong>10578</strong>.",
+        "HELGEN": "Ne vous connectez PAS à un serveur si vous êtes toujours dans la séquence d'introduction de Helgen. Assurez-vous d'abord d'avoir fui Helgen."
+      },
+      "ADDRESS": "Addresse",
+      "PASSWORD": "Mot de passe",
+      "SAVE_PASSWORD": "Enregistrer le mot de passe ?",
+      "CONNECT": "Connexion",
+      "CANCEL": "Annuler",
+      "PUBLIC_SERVERS": "Serveurs publics",
+      "ERROR": {
+        "CONNECTION": "Impossible de se connecter au serveur spécifié. Veuillez vous assurer que vous avez entré l'adresse correcte et que vous ne rencontrez pas de problèmes de réseau.",
+        "VERSION_MISMATCH": "Vous ne pouvez pas vous connecter au serveur car il ne dispose pas de la même version de Skyrim Together que vous.",
+        "INVALID_ADDRESS": "Le format de l'adresse est invalide. (Adresse:Port)"
+      }
+    },
+    "DISCONNECT": {
+      "PROMPT": "Vous êtes sûr de vouloir vous déconnecter ?",
+      "PROCEED": "Allez-y",
```

Comment:
```diff
- "PROCEED": "Allez-y",
+ "PROCEED": "Quitter",
```

"Quitter" which means "Quit / Leave" is generally the term used to proceed to quit/leave a software in french.